alert-common/src/main/java/com/synopsys/integration/alert/common/channel/message/ChannelMessageParser.java

```diff
@@ -77,14 +79,24 @@ public String createHeader(MessageContentGroup messageContentGroup) {
         return headerSeparator;
     }
 
-    public String getCommonTopic(MessageContentGroup messageContentGroup) {
-        return createLinkableItemString(messageContentGroup.getCommonTopic(), true) + getLineSeparator();
+    public String getCommonTopic(MessageContentGroup messageContent, ItemOperation nullableTopLevelAction) {
+        LinkableItem commonTopic = messageContent.getCommonTopic();
+        if (ItemOperation.DELETE.equals(nullableTopLevelAction)) {
+            commonTopic = new LinkableItem(commonTopic.getName(), commonTopic.getValue());
+        }
+        return createLinkableItemString(commonTopic, true) + getLineSeparator();
     }
 
-    public String getComponentSubTopic(ProviderMessageContent messageContent) {
-        return messageContent.getSubTopic()
-                   .map(item -> createLinkableItemString(item, true))
-                   .orElse("");
+    public String getComponentSubTopic(ProviderMessageContent messageContent, ItemOperation nullableTopLevelAction) {
```

Comment:
Why not add `@Nullable`, javax.annotation.Nullable, to the nullableTopLevelAction?